src/de/blau/android/propertyeditor/TagEditorFragment.java

```diff
@@ -708,7 +710,7 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
 					if (isChecked) {
 						tagSelected();
 					} else {
-						tagDeselected();
+						relationDeselected();
```

Comment:
I would suggest renaming everything which operates on a generic entry to "entryXY" etc otherwise this is going to cause endless confusion.